lib/staging/tls/tls.cpp

```diff
@@ -1341,4 +1390,72 @@ Client::override_t Client::default_overrides() {
     };
 }
 
+TlsKeyLoggingServer::TlsKeyLoggingServer(const std::string& interface_name, uint16_t port) {
```

Comment:
The general approach in this code is to follow MISRA guidelines and only have a single return from a function/method

Reply:
With multiple returns, the code is much easier to read. That's why I don't like this MISRA rule and wouldn't like to rewrite the constructor.

Reply:
I'm used to MISRA code I find it easier to read when there are not multiple returns. It all depends on what you are used to. I don't know what standards we are supposed to work to, however I think mixing approaches in the same file should be avoided. Happy to propose updated code if you would like.

Reply:
If you like you can propose some code :+1:  I am working on the remaining points.

Reply:
Yeah, your changes work for me :+1: